Composer/packages/client/src/components/ErrorPopup/ErrorPopup.tsx

```diff
@@ -47,7 +48,7 @@ export const ErrorPopup = (props: ErrorPopupProps) => {
     >
       <div css={consoleStyle}>{props.error}</div>
       <DialogFooter>
-        <PrimaryButton text="Ok" onClick={closeDialog} />
+        <PrimaryButton text={formatMessage('Ok')} onClick={closeDialog} />
```

Comment:
Should this be OK rather than Ok (I think the K is lowercase, but it is hard to tell in this font)?